### DISCUSSION TITLE
Is there a way to pass a route to js?

### DISCUSSION THREAD

**Discussion by realtebo** [Laravel-Backpack/community-forum#647] [Q&A (Help)] [answered]
Probably I dreamed but ... Am I wrong or there is a recently added `route` function in backpack included js ?

I would like to avoid to hardcode an url into js when doing ajax calll

**Comment by tabacitu** [comment, marked as answer]
Hey @realtebo , it was probably a dream yes 🌨️ ! We don't have such helper and AFAIK there is no `route()` helper in JS, no. 
You can use the current URL as the base, but that's just normal JS.

**Comment by pxpm** [comment]
Hey @realtebo , maybe you saw this package we use sometimes? https://github.com/Laravel-Backpack/CRUD/blob/9f494d6e3a330f849ccd10c6570622f9e7844da9/src/resources/views/crud/inc/filters_navbar.blade.php#L29 ?

**Reply by realtebo** [reply]
@pxpm  thanks.. really, It was an other package, for Laravel, not specific for bakcpack, for example: https://laravel-news.com/ziggy-named-laravel-routes-javascript
